sdk/script/account-manager.ts

```diff
@@ -0,0 +1,349 @@
+import * as base64 from "base-64";
```

Comment:
Other requested SDK improvements, such as making AccountManager the default export, and renaming all instances of 'Package' to 'Update', will be addressed in a subsequent PR.